hail/python/hailctl/dataproc/start.py

```diff
@@ -173,8 +173,11 @@ def main(args, pass_through_args):
     if args.bucket:
         conf.flags['bucket'] = args.bucket
 
-    label = safe_call('gcloud', 'config', 'get-value', 'account')
-    conf.flags['labels'] = 'creator=' + label.replace('.', '-dot-').replace('@', '-at-')  # no special chars
+    try:
+        label = sp.check_output('gcloud config get-value account', shell=True)
```

Comment:
Why are we invoking a shell here?

Reply:
the alternative doesn't work. The way gcloud picks up credentials is totally mystifying.

Reply:
oh wait, yes it does

Reply:
will fix